src/components/SignUp/ActorInfo2.tsx

```diff
@@ -199,7 +199,7 @@ const ActorInfo2: React.FC<{
               <Form.Group>
                 <CAGLabel>
                   I would also be comfortable playing roles usually played by:{' '}
-                  <PrivateLabel />
+                  <PrivateLabel className="small-size" />
```

Comment:
And then no changes should be required for the `PrivateLabel` component at all, since we aren't trying to change it's size/style - we just want to change the text label for these two fields.